roles/infra/tasks/main.yml

```diff
@@ -8,6 +8,49 @@
       }
     dest: "{{ terraform_project_path }}/backend.tf"
 
+- block:
+  - name: Set a temporary fact containing terraform_backend_config
+    set_fact:
+      _terraform_backend_config: "{{ terraform_backend_config }}"
```

Comment:
I don't understand what the actual intent/outcome here is, could you add a comment? Possibly:
- `_terraform_backend_config` gets set to the current `terraform_backend_config` ?
- `terraform_backend_config` gets set to null ?
is that correct?

Reply:
It is right yes - I've added some comments to describe the why too.